tests/test_loky_backend.py

```diff
@@ -279,6 +279,42 @@ def test_connection_queue(self):
         parent_connection.close()
         child_connection.close()
 
+    @staticmethod
+    def _test_child_env(key, queue):
+        import os
+
+        queue.put(os.environ.get(key, 'not set'))
+
+    def test_child_env_process(self):
+        import os
+
+        key = 'loky_child_env_process'
+        value = 'loky works'
+        out_queue = self.SimpleQueue()
+
+        # Test that the environment variable is correctly copied in the child
+        # process.
+        os.environ[key] = value
```

Comment:
You should probably wrap this with a try / finally block to remove the `loky_child_env_process` environment variable after the end of the test to avoid any test side effect on other tests.

Reply:
done